diff --git a/app/javascript/dashboard/components-next/sidebar/Sidebar.vue b/app/javascript/dashboard/components-next/sidebar/Sidebar.vue
index a37cde89f..4dfcae0d3 100644
--- a/app/javascript/dashboard/components-next/sidebar/Sidebar.vue
+++ b/app/javascript/dashboard/components-next/sidebar/Sidebar.vue
@@ -22,6 +22,8 @@ import SidebarAccountSwitcher from './SidebarAccountSwitcher.vue';
 import Logo from 'next/icon/Logo.vue';
 import ComposeConversation from 'dashboard/components-next/NewConversation/ComposeConversation.vue';
 
+import { useAdmin } from 'dashboard/composables/useAdmin';
+const { isAdmin } = useAdmin();
 const props = defineProps({
   isMobileSidebarOpen: {
     type: Boolean,
@@ -593,6 +595,12 @@ const menuItems = computed(() => {
     },
   ];
 });
+const filteredMenuItems = computed(() => {
+  if (isAdmin.value) {
+    return menuItems.value;
+  }
+  return isAdmin.value ? menuItems.value : menuItems.value.filter(item => !['Portals', 'Captain', 'Settings', 'Inbox'].includes(item.name));
+});
 </script>
 
 <template>
@@ -651,7 +659,7 @@ const menuItems = computed(() => {
     <nav class="grid overflow-y-scroll flex-grow gap-2 px-2 pb-5 no-scrollbar">
       <ul class="flex flex-col gap-1.5 m-0 list-none">
         <SidebarGroup
-          v-for="item in menuItems"
+          v-for="item in filteredMenuItems"
           :key="item.name"
           v-bind="item"
         />
