diff --git a/app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue b/app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue
index dd2e7a607..71762a007 100644
--- a/app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue
+++ b/app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue
@@ -224,6 +224,7 @@ export default {
     isReplyButtonDisabled() {
       if (this.isEditorDisabled) return true;
       if (this.isATwitterInbox) return true;
+      if (!this.isPrivate && !this.isConversationOpen) return true;
       if (this.hasAttachments || this.hasRecordedAudio) return false;
 
       return (
@@ -439,6 +440,9 @@ export default {
         !this.currentChat.can_reply
       );
     },
+    isConversationOpen() {
+      return this.currentChat?.status === wootConstants.STATUS_TYPE.OPEN;
+    },
   },
   watch: {
     currentChat(conversation, oldConversation) {
